### PR TITLE
Update docs with more detailed Role/Policy information

### PR DESCRIPTION
**Pull request by dustins**
*Issue #, if available:*

*Description of changes:*
We have received a significant number of inquiries about our roles and policies, particularly concerning the rationale behind specific permissions. In response, I have developed comprehensive documentation that outlines all of our policies. This initial step provides a valuable resource for our customers and can be expanded over time to address future questions.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: frontend/docs/admin-guide/security/policies/project-user-permission-boundary.md [hunk 1]
```diff
@@ -0,0 +1,437 @@
+# Project User Permission Boundary Policy
+
+---
+
+The project user permission boundary policy showcases the maximum permission set a user can have in an MLSpace project. With secure user specific roles enabled this boundary is scoped down even further to only allow users to have access to the resources their Project and Groups enable.
```
Comment (dustins) [inline frontend/docs/admin-guide/security/policies/project-user-permission-boundary.md:5]: Capitalize User and Project?

File: frontend/docs/admin-guide/security/policies/project-user-permission-boundary.md [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+    {
+      "Action": "sagemaker:AddTags",
+      "Resource": "arn:*:sagemaker:*:012345678910:*",
+      "Effect": "Allow"
+    }
+```
+
+### Statement 9
+
+These actions grant users the ability to interact with various Cloudwatch, Comprehend, Ec2, IAM, Sagemaker, and Translate operations.
```
Comment (dustins) [inline frontend/docs/admin-guide/security/policies/project-user-permission-boundary.md:164]: EC2 (capitalize C)

File: frontend/docs/admin-guide/security/policies/project-user-permission-boundary.md [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+        "translate:TranslateText"
+      ],
+      "Resource": "*",
+      "Effect": "Allow"
+    }
+```
+
+### Statement 10
+
+These actions grant users the ability to create Sagemaker endpoints but only if the User, System, and Project tags are included in the request.
```
Comment (dustins) [inline frontend/docs/admin-guide/security/policies/project-user-permission-boundary.md:209]: Capitalize E?

**Comment by dustins** [comment]
I think this looks good. I'd recommend using bedrock/claude sonnet 3.5 and for each explanation I'd run it through like this...

```
proofread and rewrite professionally...

<blurb>
```

**Review by estohlmann — APPROVED** [review]

**Pull request merged**